kernel/conversion.ml

```diff
@@ -994,6 +1011,8 @@ let () =
   in
   CClosure.set_conv conv
 
+let checked_universes = checked_universes None
```

Comment:
passing None seems dubious, seems like it should take an env and check that the quality is known like in default_evar_handler instead